AdyenCard/Components/GiftCardComponent/GiftCardComponent.swift

```diff
@@ -124,7 +125,7 @@ public final class GiftCardComponent: PartialPaymentComponent,
     internal lazy var securityCodeItem: FormTextInputItem = {
         let item = FormTextInputItem(style: style.textField)
         item.title = localizedString(.cardPinTitle, localizationParameters)
-        item.validator = NumericStringValidator(minimumLength: 3)
+        item.validator = NumericStringValidator(minimumLength: 3, maximumLength: 10)
```

Comment:
Just a curiosity: Security Code is always numerical?

Reply:
it is numerical.